services/hanke-service/build.gradle.kts

```diff
@@ -88,8 +88,7 @@ dependencies {
 	implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.7.2")
 
 	implementation("org.postgresql:postgresql")
-	implementation("org.springdoc:springdoc-openapi-kotlin:$springDocVersion")
-	implementation("org.springdoc:springdoc-openapi-ui:$springDocVersion")
+	implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.2.0")
```

Comment:
I think the springdocversion value can now be removed from the beginning of the file.